### PR TITLE
Unzip Livesync app folder

### PR DESCRIPTION
**Pull request by jasssonpet**
ping @tzraikov

### CODE REVIEW

File: build/project-template/__PROJECT_NAME__.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -326,6 +326,7 @@
 					"DEBUG=1",
 				);
 				INFOPLIST_FILE = "$(SRCROOT)/__PROJECT_NAME__/__PROJECT_NAME__-Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```
Comment (jasssonpet) [inline build/project-template/__PROJECT_NAME__.xcodeproj/project.pbxproj]: This is only in debug.

**Comment by jasssonpet** [comment]
run ci

**Comment by tzraikov** [comment]
👍

**Comment by KristinaKoeva** [comment]
👍 :cry:

**Pull request merged**